users.js

```diff
@@ -1277,6 +1277,7 @@ class User {
 					}
 					return res;
 				}
+				// @ts-ignore
```

Comment:
You should define `handler` as `AnyObject | undefined`, assuming that's the problem here?

Reply:
I'm still getting an error, users.js(1282,15): error TS7017: Element implicitly has an 'any' type because type 'never' has no index signature.